app/src/main/java/com/mobile/tandil/javabaseproject/util/ValidationEnums.java

```diff
@@ -0,0 +1,9 @@
+package com.mobile.tandil.javabaseproject.util;
+
+public enum ValidationEnums {
```

Comment:
this name should be singular `ValidationEnum`

Reply:
done